### PR TITLE
fix: Ensure correct stacking context for right click menu

### PR DESCRIPTION
**Pull request by Pytal**
## Summary

- Fix incorrect stacking context of `main.app-content` `position: static` when styles are loaded in a different order e.g. when the text app is enabled

image

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/files/src/views/FilesList.vue [hunk 1]
```diff
@@ -628,7 +628,7 @@ export default defineComponent({
 	overflow: hidden;
 	flex-direction: column;
 	max-height: 100%;
-	position: relative;
```
Comment (susnux) [inline apps/files/src/views/FilesList.vue:631]: where is this applied? I do not see any "app-content" class in this file and the styles are scoped?

Comment (Pytal) [reply apps/files/src/views/FilesList.vue:631]: It is in fact scoped yes, but it seems that it only adds the data-v-[hash] to the style declaration without requiring that the class itself is in the markup 🤔

And so it still applies:
image

**Review by skjnldsv — APPROVED** [review]

File: apps/files/src/views/FilesList.vue (hunk 1 above)
Comment (susnux) [reply apps/files/src/views/FilesList.vue:631]: weird

**Review by susnux — APPROVED** [review]

**Comment by susnux** [comment]
@Pytal backport?

**Pull request merged**